client/src/containers/MainPage.js

```diff
@@ -39,10 +37,10 @@ class MainPage extends Component {
           inputValue={location}
           handleChange={this.handleChange}
         />
-        <Button><Link to="/report">Report Incident</Link></Button>
-        <Button><Link to="/data">View Reports</Link>
-
-        </Button>
+        {/* TH: inline style on Button >> Link only till we figure out whethher the routing will be done indeed through 
+          Links or via onClick handlers */}
+        <Button style="primary"><Link to="/report" style={{ color: 'white' }}>Report Incident</Link></Button>
```

Comment:
I totally didn't catch this earlier, but it's an accessibility no-no to have links inside buttons, which this outputs into as HTML. Can we remove the button wrappings and maybe move the Button style to the Links if that's what's needed? Or else create a special "link looks like a button" class here to style these.

Ideally, the rule is Buttons should do things (e.g. onSubmit, increment counter) and Links should go somewhere (e.g. new window, new page)

Reply:
@heathercodes  and @stephanie56 - not sure what was decided (if anything) about keeping or losing the links.
Theoretically I don't see an issue an issue with a button with an onClick handler determining route (assuming the router can handle this).
The wireframes and stories do mention buttons, so if we plan on replacing them with links, we should probably doublecheck with Purvi.